applications/luci-app-dockerman/Makefile

```diff
@@ -0,0 +1,12 @@
+include $(TOPDIR)/rules.mk
+
+PKG_LICENSE:=AGPL-3.0
+PKG_MAINTAINER:=lisaac <https://github.com/lisaac/luci-app-dockerman>
+
+LUCI_TITLE:=LuCI Support for docker
+LUCI_DEPENDS:=+luci-lib-docker +docker-ce +ttyd
```

Comment:
If the required luci-lib-docker is x86_64 only, it might be good to add that restriction also here, so that casual browsers of source code will understand that easily.

Although I did not figure out why luci-lib-docker is marked x86_64 only... Not obvious to my one-minute browsing, especially as the docker-ce has been marked for (aarch64||arm||x86_64)

Reply:
Thanks for the hint. I just remembered that in the beginning docker-ce had a dependecy on x86_64. 

 I will fix this